mex/common/orcid/extract.py

```diff
@@ -0,0 +1,39 @@
+from mex.common.orcid.connector import OrcidConnector
+from mex.common.orcid.models.person import OrcidRecord
+from mex.common.orcid.transform import map_orcid_data_to_orcid_record
+
+
+def get_orcid_record_by_name(
+    given_names: str = "*", family_name: str = "*"
+) -> OrcidRecord:
+    """Returns Orcidrecord of a single person for the given filters.
+
+    Args:
+        given_names: Given name of a person, defaults to non-null
+        family_name: Surname of a person, defaults to non-null
+        **filters: Key-value pairs representing ORCID search filters.
+
+    Raises:
+        EmptySearchResultError
+        FoundMoreThanOneError
+
+    Returns:
+        Orcidrecord of the matching person by name.
+    """
+    orcid_data = OrcidConnector.get_data_by_name(
```

Comment:
when you make get_data_by_name non-static, this would become:

```suggestion
    orcid_data = OrcidConnector.get().get_data_by_name(
```